lib/graphql.ex

```diff
@@ -58,6 +58,13 @@ defmodule BridgeEx.Graphql do
       @max_attempts Keyword.get(unquote(opts), :max_attempts, 1)
       @log_options Keyword.get(unquote(opts), :log_options)
 
+      if Keyword.has_key?(unquote(opts), :max_attempts) do
```

Comment:
I was going to suggest this in the previous review, but I wasn't thinking about a way to implement it. Nice!